File: bialydunajec-ddd/application/src/main/kotlin/org/bialydunajec/ddd/application/base/query/api/dto/SocialMediaDto.kt
package org.bialydunajec.ddd.application.base.query.api.dto

data class SocialMediaDto(
        val webPageUrl: String?,
        val facebookPageUrl: String?,
        val facebookGroupUrl: String?,
        val instagramUrl: String?,
        val youTubeChannelUrl: String?
)

File: frontend-adminpanel/projects/bialydunajec-admin/src/app/shared/service/rest/dto/social-media.dto.ts
export class SocialMediaDto {
  webPageUrl?: string;
  facebookPageUrl?: string;
  facebookGroupUrl?: string;
  instagramUrl?: string;
  youTubeChannelUrl?: string;

  constructor(webPageUrl: string, facebookPageUrl: string, facebookGroupUrl: string, instagramUrl: string, youTubeChannelUrl: string) {
    this.webPageUrl = webPageUrl;
    this.facebookPageUrl = facebookPageUrl;
    this.facebookGroupUrl = facebookGroupUrl;
    this.instagramUrl = instagramUrl;
    this.youTubeChannelUrl = youTubeChannelUrl;
  }
}
